sdk-rust/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "lens_sdk"
 description = "An SDK for writing bi-directional, wasm based, LensVM transformations in Rust"
-version = "0.8.0"
+version = "0.8.1"
 edition = "2024"
 readme = "../README.md"
 repository = "https://github.com/lens-vm/lens"
```

Comment:
question: Should the repo url be updated?

Reply:
Was trying to do that yesterday in another dedicated branch - VSCode keeps complaining though even `go build` is fine.  Might ping you for ideas later if I need.